rdbms/models/database.py

```diff
@@ -2,12 +2,17 @@
 
 Delete the contents of this file and complete it with your solution.
 """
-from .database_solution import (
-    create_database_engine,
-    create_database_connection,
-    create_database_session,
-    create_model_superclass,
-    engine,
-    session,
-    Base
-)
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+import os
+from dotenv import load_dotenv
+
+ROOT = os.path.dirname(os.path.abspath(__file__))
+load_dotenv(os.path.join(ROOT, '.env'))
+URL = os.environ.get("URL")
```

Comment:
by convention the database url evironment variable identifier should be DATABASE_URL